frontend/src/components/stages/transfer_view.ts

```diff
@@ -48,36 +54,57 @@ export class TransferView extends MobxLitElement {
 
     return html`
       <stage-description .stage=${this.stage}></stage-description>
-      ${this.renderCountdown()}
-      ${this.renderOverlay()}
+
+      <div>
+        Please wait while we transfer you to the next stage of the experiment.
+        Some latency may occur as we wait for additional participants.
+      </div>
+      ${this.renderCountdown()} ${this.renderOverlay()}
     `;
   }
 
   private renderOverlay() {
-    if (!this.stage?.enableTimeout
-      || this.participantService.completedStage(this.stage.id ?? '')
-      || this.authService.isExperimenter) return;
+    if (
+      !this.stage?.enableTimeout ||
+      this.participantService.completedStage(this.stage.id ?? '') ||
+      this.authService.isExperimenter
+    )
+      return;
 
     // Temporary solution: Prevent participant from clicking to other
     // stages and resetting the transfer countdown
     return html`<div class="overlay"></div>`;
   }
 
   private renderCountdown() {
-    if (!this.countdownInterval) return;
+    if (!this.countdownInterval || !this.stage?.timeoutSeconds) return;
 
-    return html`
-      <div>Time remaining: ${this.timeRemainingSeconds} seconds</div>
-    `;
+    const minutes = Math.floor(this.timeRemainingSeconds / 60);
+    const seconds = this.timeRemainingSeconds % 60;
+
+    const timeColor = getRgbColorInterpolation(
+      '#A8DAB5', // Green
+      '#FBA9D6', // Red
+      this.timeRemainingSeconds,
+      this.stage?.timeoutSeconds
+    );
+    return html` <div>
+      If the transfer cannot be completed, the experiment will end in:
```

Comment:
We should only render this message if `this.stage.enableTimeout` is true. (Actually, we should probably not render a countdown at all if timeout is not enabled.)

Reply:
Makes sense, changed. (This was the case before, but we were checking on `countdownSeconds` which is set if `enableTimeout`).